utils/message-generator/src/main.rs

```diff
@@ -710,111 +733,123 @@ mod test {
-    //        endpoint_port: 0,
-    //        vendor: "".to_string().try_into().unwrap(),
-    //        hardware_version: "".to_string().try_into().unwrap(),
-    //        firmware: "".to_string().try_into().unwrap(),
-    //        device_id: "".to_string().try_into().unwrap(),
-    //    });
-
-    //    let frame = Sv2Frame::from_message(
-    //        setup_connection.clone(),
-    //        const_sv2::MESSAGE_TYPE_SETUP_CONNECTION,
-    //        0,
-    //        true,
-    //    )
-    //    .unwrap();
-
-    //    let frame = EitherFrame::Sv2(frame);
-
-    //    let pool_address = SocketAddr::new("127.0.0.1".parse().unwrap(), 34254);
-    //    let pub_key: EncodedEd25519PublicKey = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"
-    //        .to_string()
-    //        .try_into()
-    //        .unwrap();
-    //    tokio::time::sleep(std::time::Duration::from_secs(1)).await;
-    //    let (recv_from_pool, send_to_pool) = setup_as_downstream(pool_address, Some(pub_key)).await;
-    //    send_to_pool.send(frame.try_into().unwrap()).await.unwrap();
-    //    match recv_from_pool.recv().await.unwrap() {
-    //        EitherFrame::Sv2(a) => {
-    //            assert!(true)
-    //        }
-    //        _ => assert!(false),
-    //    }
-    //    let mut child = os_command(
-    //        "rm",
-    //        vec!["-rf", "./test/appdata/bitcoin_data/regtest"],
-    //        ExternalCommandConditions::None,
-    //    )
-    //    .await;
-    //    child.unwrap().wait().await.unwrap();
+        let frame = Sv2Frame::from_message(
+            setup_connection.clone(),
+            const_sv2::MESSAGE_TYPE_SETUP_CONNECTION,
+            0,
+            true,
+        )
+        .unwrap();
 
-    //    // TODO not panic in network utils but return an handler
-    //    //pool.kill().unwrap();
-    //    //bitcoind.kill().await.unwrap();
-    //    assert!(true)
-    //}
+        let frame = EitherFrame::Sv2(frame);
+
+        let pool_address = SocketAddr::new("127.0.0.1".parse().unwrap(), 34254);
+        let pub_key: Secp256k1PublicKey = "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72"
+            .to_string()
+            .try_into()
+            .unwrap();
+        tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+        let (recv_from_pool, send_to_pool) = setup_as_downstream(pool_address, Some(pub_key)).await;
+        send_to_pool.send(frame.try_into().unwrap()).await.unwrap();
+        match recv_from_pool.recv().await.unwrap() {
+            EitherFrame::Sv2(_a) => {
+                assert!(true)
+            }
+            _ => assert!(false),
+        }
+        // dont clean the following commented code
+        /*
+        let mut child = os_command(
+            "rm",
+            vec!["-rf", "./test/appdata/bitcoin_data/regtest"],
+            ExternalCommandConditions::None,
+        )
+        .await;
+        child.unwrap().wait().await.unwrap();
+        */
```

Comment:
why are we keeping these comments?

`//dont clean the following commented code` doesn't give any context

Reply:
because `it_test_against_remote_endpoint` is a test that I want to make it pass and not just ignore it. It is also true that there is nothing against to uncomment it and have it as "non-passing test"

Reply:
same as above. comments are not memo for you.